doc/sphinx/practical-tools/coqide.rst

```diff
@@ -5,130 +5,183 @@
 Coq Integrated Development Environment
 ========================================
 
-The Coq Integrated Development Environment is a graphical tool, to be
-used as a user-friendly replacement to `coqtop`. Its main purpose is to
-allow the user to navigate forward and backward into a Coq
-file, executing corresponding commands or undoing them respectively.
-
-CoqIDE is run by typing the command `coqide` on the command line.
-Without argument, the main screen is displayed with an “unnamed
-buffer”, and with a filename as argument, another buffer displaying
-the contents of that file. Additionally, `coqide` accepts the same
-options as `coqtop`, given in :ref:`thecoqcommands`, the ones having obviously
-no meaning for CoqIDE being ignored.
+.. todo PR: how to say that a number of things are broken?  Maybe list them
+   somewhere--doesn't have to be super detailed
+
+The Coq Integrated Development Environment (CoqIDE) is a user-friendly GUI
+for Coq. Its main purpose is to
+allow users to edit scripts and step forward and backward through Coq scripts,
+respectively executing and undoing commands and tactics.
+
+To run CoqIDE, enter `coqide` on the command line.
+If you include script file names as arguments, each is opened
```

Comment:
Do we mention that "script file" = ".v file"? We should perhaps include (.v) or something.

Reply:
I'm sure it's in the document somewhere but some repetition is fine.